strabo/templates/private/upload_images.html

```diff
@@ -24,19 +24,15 @@
             </div>
 
             <div class="form-group">
-            <label for="year">Date Taken:</label><br>
+            <label for="year">Date Taken [mm]-[yyyy]:</label><br>
             <i>Images are ordered by date when displayed.</i>
```

Comment:
What does this mean? I know this wasn't a change in this PR, but I don't think this would be very clear to a user.